test/jasmine/tests/svg_text_utils_test.js

```diff
@@ -74,7 +74,7 @@ describe('svg+text utils', function() {
         }
 
         afterEach(function() {
-            d3.select('#text').remove();
+            d3.selectAll('.text-tester').remove();
```

Comment:
the cases I added (previously and in the next commit) that test several input strings with identical output were not properly cleaned up with just `select`.